kuka_kr150_support/package.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr150_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR150-2 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 150-2 manipulators. This currently includes the KR 150-2 only.
```

Comment:
Same here. The support package is for the series, while the it only includes the `-2` variant right now.